conda_smithy/linter/hints.py

```diff
@@ -174,3 +178,24 @@ def hint_check_spdx(about_section, hints):
             "Documentation on acceptable licenses can be found "
             "[here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions )."
         )
+
+
+def hint_pip_no_build_backend(host_or_build_section, package_name, hints):
+    if host_or_build_section and any(
+        req.split(" ")[0] == "pip" for req in host_or_build_section
+    ):
+        found_backend = False
+        for backend in VALID_PYTHON_BUILD_BACKENDS:
+            if any(
+                req.split(" ")[0] == backend for req in host_or_build_section
+            ):
+                found_backend = True
+                break
+
+        if not found_backend:
+            hints.append(
```

Comment:
Since the list is not exhaustive, let's make it a lint and the setuptools issue would be fixed when we do python 3.13 build anyways.

Reply:
done!

Reply:
Sorry I mixed up `hint` and `lint`

Reply:
oh ok so I will revert my comment converting the hint to lint! thanks!